login-workflow/docs-new/README.md

```diff
@@ -0,0 +1,98 @@
+
+# Installation
+
+To install the latest version of this package, run:
+
+```shell
+npm install --save @brightlayer-ui/react-auth-workflow
+// or
+yarn add @brightlayer-ui/react-auth-workflow
+```
+
+# Integration
+
+You have a few options for using this package in your application. You can manually integrate the package into an existing project, or you can start a project using the `/example` project or by using the Brightlayer UI CLI.
+
+To integrate the package into an existing project, read our [Existing Project Integration](https://github.com/etn-ccis/blui-react-workflows/tree/master/login-workflow/docs/existing-project-integration.md) instructions. Even if you aren't starting from scratch, it may be useful for you to refer to the example project while getting started.
+
+To generate a new authentication project using the [Brightlayer UI CLI](https://www.npmjs.com/package/@brightlayer-ui/cli), run:
+```shell
+npx -p @brightlayer-ui/cli blui new --framework=react-native --template=authentication
+```
+
+To use the example project as a starting point, read our [Sample Project Integration](https://github.com/etn-ccis/blui-react-workflows/tree/master/login-workflow/docs/sample-project-integration.md) instructions.
+
+---
+## Routing
+
+We don't prescribe a routing solution, however we do recommend using [React Router](https://reactrouter.com/). For example usage details, refer to the [Routing](https://github.com/etn-ccis/blui-react-workflows/tree/master/login-workflow/docs/routing.md) documentation.
+
+## APIs
+
+More information about React Auth Workflow's exported objects and functions can found in the [API](https://github.com/etn-ccis/blui-react-workflows/tree/master/login-workflow/docs/API.md) documentation.
+
+## Error Management
+
+For information about handling error in your application, refer to our [Error Management](https://github.com/etn-ccis/blui-react-workflows/tree/master/login-workflow/docs/error-management.md) documentation.
+
+## Language Support
+
+For information about supporting multiple languages, refer to our [Language Support](https://github.com/etn-ccis/blui-react-workflows/tree/master/login-workflow/docs/language-support.md) guidelines.
+
+## Customizing the Workflow
+
+The workflow allows for some customizations. For details refer to our [Customization](https://github.com/etn-ccis/blui-react-workflows/blob/master/login-workflow/docs/customization.md) documentation.
+
+## Browser Support
+
+Brightlayer UI Login Workflow will work with any modern browser. For details refer to our [Browser Support](https://brightlayer-ui.github.io/development/frameworks-web/react#browser-support) documentation.
+
+---
+
+# Contributors
+
+To work on this package as a contributor, first clone down the repository:
+
+```shell
+git clone https://github.com/etn-ccis/blui-react-workflows
```

Comment:
I think this is fine for what it is. This may be reworked later, but for now, the cloning script is fine. I believe we cover forks in our contribution docs as well so end-users should be aware of forking through that.